src/sources/video_card.test.ts

```diff
@@ -0,0 +1,36 @@
+import getVideoCard from './video_card'
+
+describe('Sources', () => {
+  describe('videoCard', () => {
+    const videoCard = getVideoCard()
+    it('to be defined', () => {
+      expect(videoCard).toBeDefined()
```

Comment:
Do you expect a defined value in [all the browsers that we support](https://github.com/fingerprintjs/fingerprintjs/blob/master/docs/browser_support.md)? If no, please make an expectation based on the current browser (example: https://github.com/fingerprintjs/fingerprintjs/blob/b4c5d0e2193355661529ada99c546ae12bba71c3/src/sources/audio.test.ts).

Reply:
After some testing. older browsers (ex IE11) return a generic response such as :
```
"videoCard": {
    "value": {
        "vendor": "Microsoft",
        "renderer": "Microsoft Basic Render Driver"
    }
    "duration": 3
}
```

And I added a condition to verify access to webgl or experimental-webgl and verify they have 'getExtension'.
else return undefined.

Reply:
Since a defined value is expected in every browser, please add an assertion that checks that the returned value is defined

Reply:
An undefined value is returned if gl or getExtension or renderer_info isn't available in the webgl canvas.
I don't know if all browsers support this.
I couldn't test on old android since I don't have one and browserstack doesn't work for me for an unknown reason.

Reply:
I've run the Jasmine tests on you branch on BrowserStack and have found out that the value is `undefined` only in Firefox 52. According to https://developer.mozilla.org/en-US/docs/Web/API/WEBGL_debug_renderer_info, the value should be defined in Firefox 53+. So please add the following assertion to the test:

```js
if (!isGecko() || (getBrowserMajorVersion() ?? Infinity) >= 53) {
  expect(videoCard).toBeTruthy()
}
```

Reply:
Done !